src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -119,28 +128,39 @@ public void canLoadFeedWithBadDates () {
                 }
             )
         );
+        ErrorExpectation[] errorExpectations = ErrorExpectation.list(
+            new ErrorExpectation(NewGTFSErrorType.MISSING_FIELD),
+            new ErrorExpectation(NewGTFSErrorType.DATE_FORMAT),
+            new ErrorExpectation(NewGTFSErrorType.DATE_FORMAT),
+            new ErrorExpectation(NewGTFSErrorType.DATE_FORMAT),
+            new ErrorExpectation(NewGTFSErrorType.REFERENTIAL_INTEGRITY)
+        );
         assertThat(
             "Integration test passes",
-            runIntegrationTestOnFolder("fake-agency-bad-calendar-date", nullValue(), expectations),
+            runIntegrationTestOnFolder("fake-agency-bad-calendar-date", nullValue(), expectations, errorExpectations),
             equalTo(true)
         );
     }
 
     /**
-     * Tests that a GTFS feed with overlapping block trips will record the appropriate error.
+     * Tests that a GTFS feed with errors is loaded properly and that the various errors were detected and stored in the
+     * database. Note: the errors should be listed in order that they are expected to be encountered. Check out
+     * {@link JdbcGtfsLoader#loadTables} to see the order in which tables are loaded, {@link Feed#validate()} to see the
+     * order in which validators are called (trip validator order can be found in
+     * {@link com.conveyal.gtfs.validator.NewTripTimesValidator}).
      */
     @Test
-    public void canLoadFeedWithOverlappingTrips () {
-        PersistenceExpectation[] expectations = PersistenceExpectation.list(
-            new PersistenceExpectation(
-                new ErrorExpectation[]{
-                    new ErrorExpectation("error_type", NewGTFSErrorType.TRIP_OVERLAP_IN_BLOCK.toString())
-                }
-            )
+    public void canLoadFeedWithErrors () {
+        PersistenceExpectation[] expectations = PersistenceExpectation.list();
+        ErrorExpectation[] errorExpectations = ErrorExpectation.list(
+            new ErrorExpectation(NewGTFSErrorType.FARE_TRANSFER_MISMATCH, "fare-02"),
+            new ErrorExpectation(NewGTFSErrorType.FREQUENCY_PERIOD_OVERLAP, "freq-01_09:00:00_to_10:00:00_every_10m00s"),
+            new ErrorExpectation(NewGTFSErrorType.FREQUENCY_PERIOD_OVERLAP, "freq-01_09:30:00_to_10:30:00_every_15m00s"),
```

Comment:
I think it'd be good to add 2 more error expectations here to cover all possible overlaps:

Key: 

`|`: start or end of frequency
`-`: active time of frequency
`_`: frequency not active

A covers B:

A: `|---------|`
B: `___|--|____`

B covers A

A: `___|--|____`
B: `|---------|`

A starts before B starts, but A ends after B starts and A ends before B ends:

A: `_|----|____`
B: `____|-----|`

A starts after B starts, but A also starts before B ends

A: `____|-----|`
B: `_|----|____`